tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -677,6 +677,10 @@ private static void testPixelCapture(boolean primitiveArg)
 
     @Test
     public void testPixelCaptureAverage() throws Exception {
+        if (PlatformUtil.isWindows() && Screen.getPrimary().getOutputScaleX() > 1) {
+            //Mark test as unstable for HiDPI scale greater than 100%
+            Assume.assumeTrue(Boolean.getBoolean("unstable.test"));
```

Comment:
Please add the BugID of the test failure bug. See [SwingNodeMemoryLeakTest.java#L88](https://github.com/openjdk/jfx/blob/fb6788a30160425a436a530a9cb511f633fb642b/tests/system/src/test/java/test/javafx/embed/swing/SwingNodeMemoryLeakTest.java#L88) for example.